app/models/quiz_answer.rb

```diff
@@ -1,5 +1,17 @@
 # frozen_string_literal: true
 
 class QuizAnswer < ApplicationRecord
+  include ::V2::Image::ImageService
+
+  has_one_attached :answer_image_url_s3
```

Comment:
Sorry this is really pedantic but do you think we remove "url" and "link" from these _s3 names. I feel like they aren't really links/url any more given they are s3 objects.

Reply:
100% agree!